gmime/part.go

```diff
@@ -34,6 +34,12 @@ func (p *Part) IsAttachment() bool {
 	return gobool(C.g_mime_part_is_attachment((*C.GMimePart)(unsafe.Pointer(p.gmimePart))))
 }
 
+// Filename retrieves the filename of the part
+func (p *Part) Filename() string {
```

Comment:
what do you get back when there is no filename associated with part? can we test?

Reply:
Its just an empty string (returned as null)